### PR TITLE
Qt update to 6.6.1, emscripten to 3.1.37

### PR DESCRIPTION
**Pull request by dirkjanfaber**
Update Qt version to 6.6.1 and emscripten to 3.1.37. Made the patching more generic, install `yq`, update and install python is now done after the cache check (should speed things up). Skip steps when using cached version.

Have packing + patching html and `js` files into a zip as separate step. The patchings are showing the Victron logo and to set the default argument for connecting to the mqtt broker. And to set the qtAppInstance as window (to make it work via VRM). Remove `qtloader-controller.js` (as it is no longer needed for 6.6.1).

Also see issue #734

### CODE REVIEW

**Review by chriadam — APPROVED** [review]

File: .github/workflows/build-wasm.yml [hunk 1]
```diff
@@ -7,34 +7,37 @@ on:
 jobs:
   build-wasm-file:
     env:
-      EMSCRIPTEN: 3.1.25
-      QT_VERSION: 6.5.2
+      EMSCRIPTEN: 3.1.37
+      QT_VERSION: 6.6.1
```
Comment (DanielMcInnes) [inline .github/workflows/build-wasm.yml:11]: Should we change the required Qt version in CMakeLists.txt as well?

**Review by DanielMcInnes — APPROVED** [review]
lgtm, just 1 question

File: .github/workflows/build-wasm.yml (hunk 1 above)
Comment (dirkjanfaber) [reply .github/workflows/build-wasm.yml:11]: AFAIK the non-webassembly build still uses 6.5.2 for building. Once that has changed to 6.6.1 I'd change the `CMakeLists.txt`.

Comment (DanielMcInnes) [reply .github/workflows/build-wasm.yml:11]: lgtm

**Pull request merged**